common/src/main/java/earth/terrarium/hermes/api/defaults/TextTagElement.java

```diff
@@ -18,6 +20,7 @@ public abstract class TextTagElement implements TagElement {
     protected @Nullable Boolean strikethrough;
     protected @Nullable Boolean obfuscated;
     protected boolean centered;
```

Comment:
mark centered as deprecated if its not used for anything other than setting `align`

Reply:
Done